addons/vol-slider/userstyle.css

```diff
@@ -0,0 +1,29 @@
+.sa-volume > * {
+  vertical-align: middle;
+  margin-top: 6px;
+}
```

Comment:
I think we could add a `margin-left` here, so the volume icon isn't so close to the stop button.

Also, we might need to support RTL here, and put the margin in the other side. Not sure if other addons that add their UI elements above the stage take RTL into account for margins. If they do, this addon should too.

Reply:
I forgot about RTL...

Reply:
We'd need to check if other similar addons take it into account.
If not, we can proceed. But we need to stay consistent with other addons that use the editor stage head